src/components/grapher/Grapher.vue

```diff
@@ -207,6 +222,14 @@ export default Vue.extend({
       const currentSS: StuntSheet = this.$store.getters.getSelectedStuntSheet;
       return currentSS.stuntSheetDots;
     },
+    dotLabels(): string[] {
+      const dotLabels = this.$store.getters.getDotLabels;
+      const numDots = this.$store.getters.getSelectedStuntSheet.stuntSheetDots
+        .length;
+      return [...Array(numDots).keys()].map((i) => {
+        return dotLabels && i < dotLabels.length ? dotLabels[i] : i.toString();
+      });
```

Comment:
Can you use `dot.dotLabelIndex`? That's what `dotLabelIndex` is intended to be used for https://github.com/calband/calchart-redesign/blob/master/src/models/StuntSheetDot.ts#L22

```suggestion
      return this.$store.getters.getSelectedStuntSheet.stuntSheetDots.map((dot, index) => {
        return dotLabels && dot.dotLabelIndex < dotLabels.length ? dotLabels[dot.dotLabelIndex] : index.toString();
      });
```

Reply:
Sounds good.  Addressed!